applications/concept-cat/src/main/java/no/ccat/service/RDFToModelTransformer.java

```diff
@@ -263,12 +280,12 @@ private ContactPoint extractContactPoint(Resource resource) {
 
             Statement phoneStatement = contactPointResource.getProperty(VCARD4.hasTelephone);
             if (phoneStatement != null) {
-                String parsedPhoneNumber = parseURIFromStatement(phoneStatement);
+                String parsedPhoneNumber = getOnlySchemeSpesificPartOfURIFromStatement(phoneStatement);
```

Comment:
Does this change belong to the omfang-feautre? If not, maybe move to another PR?

Reply:
Ja nå ble det forskellige blokker med kode som bruker URI parse funksjonen, på litt forskjellig måte, så jeg splittet den i 3, en grunndel, og to forskjellige metoder som bruker grundelen, med beskrivende navn.